ui/app/common/pnc-client/resources/Environment.js

```diff
@@ -31,7 +31,7 @@
     'restConfig',
     'ENVIRONMENT_PATH',
     function($resource, restConfig, ENVIRONMENT_PATH) {
-      var ENDPOINT = restConfig.getPncUrl() + ENVIRONMENT_PATH;
+      var ENDPOINT = restConfig.getPncRestUrl() + ENVIRONMENT_PATH;
```

Comment:
I guess `Environment.js` file is completely refactored right now? so it should be renamed to `EnvironmentResource.js` (including factory method name and its usages).